projects/js-packages/charts/src/charts/pie-chart/stories/donut.docs.mdx

```diff
@@ -1,11 +1,11 @@
 import { Meta, Canvas, Story, Source } from '@storybook/addon-docs/blocks';
 import * as DonutStories from './donut.stories';
 
-<Meta title="JS Packages/Charts/Types/Donut Chart" of={ DonutStories } />
+<Meta title="JS Packages/Charts Library/Charts/Donut Chart" of={ DonutStories } />
```

Comment:
Do we consider donut chart to be a chart primitive or a composition? How about pie semi circle chart? They both use the pie chart as their base.

Reply:
Oops nevermind I see you mentioned donut chart already :) I should read the comments before I read the code! 😆

Reply:
Yeah essentially we think that consumers will see them as different chart types